src/frontend/handlers.go

```diff
@@ -48,7 +48,7 @@ var (
 	plat platformDetails
 )
 
-var validEnvs = []string{"local", "gcp", "azure", "aws", "onprem"}
+var validEnvs = []string{"local", "gcp", "azure", "aws", "onprem", "alibaba"}
```

Comment:
Could you also please add alibaba as an option here? Thanks! 
https://github.com/GoogleCloudPlatform/microservices-demo/blob/master/kubernetes-manifests/frontend.yaml#L69

Reply:
done